### ISSUE TITLE
Discworld - Missing Files (ScummVM)

### ISSUE THREAD

**Issue by alligatorshoes** [legluondunet/MyLittleLutrisScripts#37]
Followed your Discworld (ScummVM) lutris script and it all installed perfectly, thank you. However when trying to run the window would pop up and then close again immediately. Lutris debug showed the following:

> MILES-ADLIB: timbre file not found (may be called FAT.OPL, MIDPAK.AD, SAMPLE.AD or SAMPLE.OPL, may be in a subdirectory)!

This was fixed by copying the SAMPLE.AD and SAMPLE.OPL file(s) from the mounted CD/drivers folder into the root of the game installation.

**Comment by legluondunet** [comment]
Hello @alligatorshoes ,
I just tried this installer script and I can reproduce your issue:

`Initial process has started with pid 428562
Game is considered started.
WARNING: SDL mixer output buffer size: 512 differs from desired: 1024!
WARNING: Unrecognized graphics mode 'normal'. Switching to default mode!
MILES-ADLIB: timbre file not found (may be called FAT.OPL, MIDPAK.AD, SAMPLE.AD or SAMPLE.OPL, may be in a subdirectory)!
User picked target 'dw' (gameid 'dw')...
  Looking for a plugin supporting this gameid... Tinsel
  Starting 'Discworld'
Game is considered exited.
Initial process has exited.
All children have exited.
Exit with returncode 256`

My bug, I thought this folder was useless and I deleted it in the end of the script.
It's now fixed.
 thank you for your report.
